tableschema/model.py

```diff
@@ -158,7 +158,7 @@ def _expand(self, schema):
 
     @staticmethod
     def _type_map():
-        """Map a JSON Table Schema type to a JTSKit type class."""
+        """Map a Table Schema type to a JTSKit type class."""
```

Comment:
Seeing as you started renaming first :), this still has the very first name - JTSKit.